docs/get-started/cli.md

```diff
@@ -4,4 +4,219 @@ title: On your local machine
+
+## Before you get started
+
+To create a Pack you will need a Coda account, with [Doc Maker access](https://help.coda.io/en/articles/3388781-members-and-roles) in your workspace. If you're new to Coda, [sign up](https://{{coda.domain}}/signup) for a free account and you'll automatically be made a Doc Maker in your personal workspace.
+
+Make sure you have `node` and `npm` available on your machine. These two tools often come bundled together, and more information on how to install them is available in the [npm docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+The instructions below assume some familiarity with the terminal / command prompt. If you aren't used to using this interface consult the help material for your operating system.
+
+## Set up the Pack structure
+
+1. Create a directory for your new Pack.
+
+    ```sh
+    mkdir my-pack
+    cd my-pack
+    ```
+
+1. Install the Packs SDK.
+
+    ```sh
+    npm install @codahq/packs-sdk
+    ```
+
+    The Packs SDK includes both the `coda` CLI as well as the libraries and type definitions needed to build Packs.
+
+1. Create the file structure for your Pack.
+
+    ```sh
+    npx coda init
+    ```
+
+
+The `coda init` command creates the basic skeleton of a Pack based off of our [template Pack](https://github.com/coda/packs-examples/tree/main/examples/template). It's not required, but it's a fast way to get started.
+
+Your directory should now contain the following files:
+
+* `formulas.ts` - A place to define the Coda formulas, including those that power sync tables and column formats.
+* `manifest.ts` - Defines the metadata and contents of your Pack.
+* `node_modules` - The dependencies downloaded from NPM (standard for Node.js projects).
+* `package-lock.json` - The versions of the dependencies downloaded from NPM (standard for Node.js projects).
+* `package.json` - The project's dependencies from NPM (standard for Node.js projects).
+* `schemas.ts` - A place to define the schemas (structured data types) used by your Pack.
+* `types.ts` - A place to define TypeScript types for the data used by your Pack.
+
+## Add functionality to you Pack
+
+Now that you've got the basic structure of a Pack it's time to customize it and add some functionality. In this tutorial you'll be creating a simple "Hello World" Pack with a single formula.
+
+1. Edit `formulas.ts` to include the definition of a new "Hello" formula:
+
+    === "formulas.ts"
+        ```ts hl_lines="4-7 10-34"
+        --8<-- "hello_cli/formulas.ts"
+        ```
+
+1. Edit `manifest.ts` to set the formula namespace to `HelloWorld`:
+
+    === "manifest.ts"
+        ```ts hl_lines="8"
+        --8<-- "hello_cli/manifest.ts"
+        ```
+
+    !!! note
+        The value in `formulaNamespace` isn't normally visible to the user, as Coda prefers to show the Pack's icon instead. However when a formula is copied from the formula editor and pasted as plain text the namespace is used instead of the icon.
```

Comment:
The plan is to fully remove formulaNamespace from the SDK, and we're actually finally quite close to doing that. And external users will never actually see a namespace, since we're only going to allow installing user-created packs in docs that have the icon functionality. When pasting a formula as plain text in a doc using the icon, the text is actually the pack name and not the namespace.

So I guess we should find the least awkward way to say that this field is deprecated (in a brand new product...) and its value is irrelevant. The pack builder does a decent job obfuscating it because it just provides a default namespace without the maker needing to know about it.

Reply:
Definitely makes sense as a general approach. My main thing is I'd like to frame it in a way so that the maker doesn't get worried about having to understand this concept as being part of pack development, when it's basically irrelevant to their work and will disappear soon. In this case I think it's appropriate to talk about the future.

Maybe can say something like, "In older Coda docs, the namespace is used to write a formula using the formula builder, e.g. MyNamespace::MyFormula. In most docs, and soon all docs, the namespace will no longer be needed, and the formula builder will display the Pack's icon instead. Once all Coda docs have this behavior, you will no longer need to specify a namespace in the Pack definition at all."

Reply:
As per our offline discussion, I've removed mention of the `formulaNamespace` field altogether and don't instruct developers to change the value.